platform.md

```diff
@@ -566,6 +567,10 @@ Usage:
       - All run-image layers SHALL be preserved
       - All run-image config values SHALL be preserved unless this conflicts with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - MUST contain a layer containing all buildpack-provided standardized Bill of Materials (BOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md)
+      - `<layers>/BOM/<buildpack-id>/launch.bom.<ext>` MUST contain the buildpack-provided `launch` BOM
+      - `<layers>/BOM/<buildpack-id>/<layer-id>/launch.bom.<ext>` MUST contain the buildpack-provided layer BOM if `<layer-id>` is a `launch` layer
```

Comment:
```suggestion
      - `<layers>/bom/<buildpack-id>/launch.bom.<ext>` MUST contain the buildpack-provided `launch` BOM
      - `<layers>/bom/<buildpack-id>/<layer-id>/launch.bom.<ext>` MUST contain the buildpack-provided layer BOM if `<layer-id>` is a `launch` layer
```

Reply:
Fixed in latest commit (went with `<layers>/sbom` per sub sync discussion)